.github/workflows/schema.yml

```diff
@@ -28,4 +28,4 @@ jobs:
 
         - name: Test Schema
           run: |
-            pixi run -e default test-schema
+            pixi run -e schema test-schema
```

Comment:
How about also running fmt-schema + generate-schema in CI and checking the git diff?

Reply:
That is actually a good idea!

Reply:
https://github.com/prefix-dev/recipe-format/blob/ba04de9f7dd03573f979ffa80413c1e814cb669c/.github/workflows/ci.yml#L16-L30 😄  while we're at it we can also add ruff linting

Reply:
Done!